src/debuggers/natvis/magnum.natvis

```diff
@@ -101,4 +161,24 @@
       </ArrayItems>
     </Expand>
   </Type>
+  <!-- Magnum::Math::Deg< Float > -->
+  <Type Name="Magnum::Math::Deg&lt;float&gt;">
+    <DisplayString IncludeView="simple">{_value,g}° ({_value * 0.0174532924,g} radians)</DisplayString>
```

Comment:
Same here and below, just a `<!-- pi/180 -->` etc. Hmm, or maybe it could be typed out in full, i.e. `_value* (3.14whatever/180.0)`? Is that supported in this context or you can do that just in the `Intrinsic`?

Reply:
Yes, it is supported. It would be clearer to do so 👍

Reply:
I went with typing it all out route.